lib/jsdom/living/helpers/colors.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+
+exports.computeColor = maybeColor => {
+  maybeColor = maybeColor.trim();
+
+  // sRGB?
+  const srgbRegex = /^rgb *\( *(\d{1,3}) *, *(\d{1,3}) *, *(\d{1,3}) *\)$/;
+  if (srgbRegex.test(maybeColor)) {
+    const [, r, g, b] = srgbRegex.exec(maybeColor);
+    return `rgb(${r}, ${g}, ${b})`;
+  }
+
+  // hex?
+  const shorthandHexRegex = /^#([a-f\d])([a-f\d])([a-f\d])$/;
+  maybeColor = maybeColor.replace(shorthandHexRegex, (_, r, g, b) => {
+    return r + r + g + g + b + b;
+  });
+  const hexRegex = /^#([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/;
+  if (hexRegex.test(maybeColor)) {
+    let [, r, g, b] = hexRegex.exec(maybeColor);
+    r = parseInt(r, 16);
+    g = parseInt(g, 16);
+    b = parseInt(b, 16);
+    return `rgb(${r}, ${g}, ${b})`;
+  }
+
+  // definded keyword?
+  if (exports.keywords[maybeColor]) {
+    const [r, g, b] = exports.keywords[maybeColor];
+    return `rgb(${r}, ${g}, ${b})`;
+  }
+
+  return "";
+};
+
+// https://www.w3.org/TR/css-color-3/#svg-color
+exports.keywords = {
```

Comment:
Why is this exported?

Reply:
I was originally going to handle the color computation elsewhere, but I decided against that and simply didn't remove the export. I'll do that now.